@navikt/aksel-stylelint/src/deprecations.ts

```diff
@@ -68,4 +68,8 @@ export const deprecations: DeprecatedList = [
     ],
     message: "These classes had no effect and were removed in v7.1.0",
   },
+  {
+    classes: ["navds-list--nested", "navds-list__item-content"],
```

Comment:
Should we include this even though it seems like nobody have used these classes seems in CSS?

Reply:
Doesn't hurt i guess 🤷